app/posts/register-trainee-teachers/2020-10-06-beta-prototype-research-round-2.md

```diff
@@ -6,25 +6,313 @@ related:
   items:
   - text: Research playback
     href: https://docs.google.com/presentation/d/18ihO7djCDhPWZ_neSsciKrGEN5SFpyEb4u6pL6pnELE
+  - text: Research plan
+    href: https://docs.google.com/spreadsheets/d/1F1wl_VnntLwbOj5KEQb7D02X3T8w2CEeRMx9lehIywY
   - text: Research recordings
     href: https://drive.google.com/drive/folders/1x0SlQ-q8YY5N1Q4IWhSpDXy03DzGo4Ty
+  - text: User needs
+    href: https://trello.com/b/5WZeE54R/register-trainee-teachers-user-needs-board
 ---
+{% from "figure/macro.njk" import appFigure with context %}
 
-Our intention in the second round of research was to answer the following questions:
+Research completed: 27 to 29 October 2020 
 
-* Are there any other needs specific to the provider-led postgraduate route?
-* What are our users key behaviours and needs around data collection and input? 
-* What are the needs around inputting large numbers of trainees?
-* How do users handle alternative assessment outcomes to a QTS recommendation?
-* Do our users want to manage data on individual or on cohort basis and how is a cohort characterised?
-* What is understood about “allocations”?
-* What are their habits around updating records due to withdrawals and deferrals? 
+
+Our second round of research focused mainly on testing the usability of our service when inputting initial teacher training (ITT) data for the provider-led postgraduate route. 
+
+Our intention was to answer the following questions:
+
+* What are our users’ key behaviours and needs around inputting large numbers of trainees?
+* What is their understanding of “allocations”?
+* What are the different scenarios around deferrals and withdrawals?
+* What happens if a trainee fails? 
+* Is our guidance useful and do users understand the next steps around tracking TRN and QTS?
+* Do our users want to manage data on an individual or cohort basis and how is a “cohort” characterised?
+* How do users currently handle alternative assessment outcomes to a QTS recommendation? 
 
 ## Who we tested with
 
+We tested with these providers:
+
 * Associated Merseyside Partnership (AMP) SCITT
-* Essex Primary SCITT
 * Partnership London SCITT
-* Suffolk & Norfolk SCITT
-* The Pimlico London SCITT
 * University College Birmingham
+* Essex Primary SCITT
+* The Pimlico London SCITT
+* Suffolk and Norfolk SCITT
```

Comment:
As this is public, maybe we should just use the type of provider instead of naming them?